ReadAlongs/Studio
# Bring the bundled web-component up to date

### PR Goal? 

update the long stale web-component offline bundles, and current web-component version.

### Fixes? 

Fixes #219

### Feedback sought? 

just make sure it works OK

### Priority? 

normal

### Tests added? 

n/a

### How to test? 

run `cd test; readalongs align -o html data/ej-fra.txt  data/ej-fra.m4a -l fra delme` and see that the single-file HTML in delme has the gear menu (i.e., the latest web-component version) and `index.html` asks for `web-component@1.4.x`.

### Confidence? 

high

### Version change? 

We're long overdue for a release, not for this PR but for previous changes, with at least a minor bump.

Significant changes:
 - we dropped Python 3.7 and added support for 3.12
 - we added the readme in the generated directory
 - Most changes were really in the web-component, not here.

Even though dropping Python 3.7 is a breaking change, I'd be inclined to just bump this project to 1.1, not 2.0, because is really hasn't changed much.
But I don't mind going to 2.0 if we want to be strict with the breaking=>major bump semantics.
